README.md

```diff
@@ -34,16 +34,40 @@ singularity remote login
 ```
 
 ### Create singularity images. 
-Running `build_container.sh` requires sudo priviledges.
+
+Note: The containers that need to be built locally require sudo privileges.
+
 ```
-cd container && sudo bash -i build_container.sh && cd ..
+cd container
+sudo make build_local_containers
+make download_remote_containers
```

Comment:
Got the following error when running `make download_remote_containers`:
```
tee: .install.log: Permission denied
--------------------------------------------------------------------------------
Makefile:84: recipe for target 'abritamr.sif' failed
make: *** [abritamr.sif] Error 1
```

Reply:
Yeah, that seems to make sense. Trying it locally now ...